src/server/api/routes/users.router.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const usersController = require('../controllers/users.controller');
+
+/**
+ * @swagger
+ * /user:
+ *  post:
+ *    tags:
+ *    - Users
+ *    summary: Create a user
+ *    description:
+ *      Will create a user.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: user
+ *        description: The module to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - full_name
+ *            - email
+ *            - address
+ *            - zipcode
+ *            - city
+ *            - country
+ *            - created_at
+
+ *          properties:
+ *            full_name:
```

Comment:
Here you should insert a user props instead of "string".
Remember we test in swagger, not Postman